### DISCUSSION TITLE
v3.1.4-beta.9 Was wurde geändert?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2340] [Q&A]
Keine Einträge vorhanden, wo man schauen könnte.

**Comment by Zefau** [comment]
nur der technische build Workflow

**Comment by mcuiobroker** [comment]
Es gibt aber keine Dateien-> unterschiedliche Versionen für Android / Windows und Browser?
![image](https://user-images.githubusercontent.com/84818934/234075443-c5d91bbb-7ba2-48c4-acb6-f9a6b2edc69a.png)

**Reply by Zefau** [reply]
Nein, das habe ich abgebrochen. Habe was anderes automatisiert. Ist aber was internes, was mir repetitive Arbeit abnimmt, aber keine Auswirkungen auf Enduser hat.
